src/main/drivers/bus_i2c.h

```diff
@@ -53,14 +44,16 @@ typedef enum I2CDevice {
 #define I2C_ADDR7_MIN       8
 #define I2C_ADDR7_MAX       119
 
-typedef struct i2cConfig_s {
-    ioTag_t ioTagScl[I2CDEV_COUNT];
-    ioTag_t ioTagSda[I2CDEV_COUNT];
-    bool overClock[I2CDEV_COUNT];
-    bool pullUp[I2CDEV_COUNT];
-} i2cConfig_t;
+typedef enum I2CDevice {
+    I2CINVALID = -1,
+    I2CDEV_1   = 0,
+    I2CDEV_2,
+    I2CDEV_3,
+    I2CDEV_4,
+} I2CDevice;
```

Comment:
You've moved the position of this in the file. To minimise the changes in the PR, can you just leave it in it's original position.